core/src/labels/labelMesh.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "gl/typedMesh.h"
+#include "labels/label.h"
+
+#include <memory>
+#include <vector>
+
+namespace Tangram {
+
+class LabelMesh : public TypedMesh<Label::Vertex> {
+public:
+    LabelMesh(std::shared_ptr<VertexLayout> _vertexLayout, GLenum _drawMode);
+
+    virtual ~LabelMesh();
+
+    void addLabel(std::unique_ptr<Label> _label);
+
+    std::vector<std::unique_ptr<Label>>& getLabels() {
+        return m_labels;
+    }
+
+protected:
+    std::vector<std::unique_ptr<Label>> m_labels;
```

Comment:
What do you think of using a vector of objects instead of pointers here; mostly to force memory contiguity? Also reserving the containers at the beginning of tile building process with worst size to reduce copies all around while adding labels. We would need either different containers (for `spriteLabel` & `textLabel`) or a template (not for the best).